Add Aptos definition to landing page, begin to emphasize our ecosyste… (#5637)
aptos-labs/aptos-core: developer-docs-site/docs/index.md

# Aptos Developer Documentation

> Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." We at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 

Reviewer: This is technically owned by the Foundation. We just support it as the Labs. So we should make sure this is reflected as such. Let's make an issue here to follow up and ensure we respect the Foundation appropriately.

Author: > respect the Foundation

Issue filed and assigned to us:
https://github.com/aptos-labs/aptos-core/issues/5773

> Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." We at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 

Reviewer: ```suggestion
Welcome! Aptos is a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." This site is here to help you grow a [web3 ecosystem project](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 
```

> Welcome! At Aptos Labs, we are building a Layer 1 for everyone. In the [Ohlone language](https://en.wikipedia.org/wiki/Ohlone_languages), ["Aptos"](https://en.wikipedia.org/wiki/Aptos,_California) means "The People." And we at Aptos Labs are here to help you grow a [web3 ecosystem](https://github.com/aptos-foundation/ecosystem-projects) that benefits the entire world through easier development, more reliable services, faster transactions, and a supportive, decentralized family. 

Reviewer 2: imo just: "We at", not "And we at"

Author: Done!

> This documentation will help you develop applications for the Aptos blockchain, run nodes, and be a part of the blossoming Aptos community. This documentation covers both basic and advanced topics. Here you will find concepts, how-to guides, quickstarts, tutorials, API references, code examples, release notes, and more.

Reviewer 2: Awesome

Author: Thank you!

* [Forum](https://forum.aptoslabs.com/)
* [Medium](https://medium.com/aptoslabs)
* [Telegram](https://t.me/aptos_official)

> * [Twitter](https://twitter.com/AptosLabs)

Reviewer: Twitter should be updates to Foundation.

> * [Twitter](https://twitter.com/AptosLabs)

Reviewer: ```suggestion
* [Twitter](https://twitter.com/Aptos_Network)
```
